### ISSUE TITLE
1.1 Material Rotation Schedule - on behalf of Josh Jetto

### ISSUE THREAD

**Issue by zctinker** [gryphonshafer/Quizzing-Rule-Book#104]
Current Rule: John 1-21

Proposed Rule: John 1-7:52; John 8:12-21:25
Rationale: The story of the woman caught in the act of adultery from John 7:53-8:11 is not found in the earliest manuscripts of the Gospel of John. The scholarly consensus (among evangelical scholarship as well) is that it was not written by John. Since it is like 99.9999% certain that is not a part of his writing, it should not be regarded as Scripture, and I don’t think we should encourage students to memorize it as though it were. Not only does the insertion of this story break up the flow of the text of John’s Gospel and of Jesus’ speaking at the Feast of Tabernacles, but it has also been the source of some poor practical theology (how we should handle sin in the church) as well as having been used by some as the justification of themselves for their engagement in sexual immorality (since, as is it is sometimes said, “Jesus didn’t condemn the woman caught in adultery”). The bottom line is this: It is not Scripture. Anytime we put something that is not Scripture at the level of Scripture, false teaching, false living, and a false view of God ensue.

**Comment by jttower** [comment]
I hear what you are saying, Josh, but I think the bottom line is it IS Scripture since it is included in the Bible.  Since potentially some people might use this passage for questionable theology, that gives us as coaches a chance to help students understand what Jesus meant when he didn't condemn her, but asked her to leave her life of sin.  (Anyone using this passage to justify sexual immorality is leaving out the rest of that verse.) All Scripture needs to be taken together, using Scripture to interpret Scripture, and that can be done with this passage.  The fact is we don't have a leg to stand on if we are looking to condemn a sinner, since we are all sinners.  There are plenty of places in Scripture that talk about how to handle sin in the church and others that talk about sexual immorality.  There are certainly plenty of passages that we study in BQ that can be taken out of context and used to build an incorrect theology, but that doesn't mean we leave them out.  Our quizzers need to know how to correctly handle the Scriptures, especially the "tricky" passages.  For these reasons, I don't see a reason to take it out of our study of John since it is included in Canon. In fact, for the reasons I listed, I think we should leave it in so that our students can be equipped in their interpretation of Scripture.

**Comment by JoshJetto** [comment]
There is a reason that every modern translation brackets off this section (as well as the longer ending of Mark) and says, "the earliest and most reliable manuscripts do not contain these passages." It is because virtually every person who works with the multitude of Greek manuscripts of the New Testament agree that these passages were not written by the Apostolic Gospel writers, meaning that they are actually not the inspired word of God but additions to that inspired word. They only continue to be included in Bibles (though bracketed off) because they were included in the later manuscripts and incorporated into the Greek New Testament that served as the basis for the King James Version, and they have a very long "tradition" in the church. Otherwise, they would be relegated to footnotes like the other "missing" verses in the NIV, which have been found to be later additions to the text. The reason I would advocate not having students memorize these verses in John as Scripture is because the textual evidence points very strongly to them not being original Scripture.

**Comment by gryphonshafer** [comment]
We already skip portions of Scripture that are in the earliest manuscripts, like the genealogy in Matthew. I don't see the problem with skipping portions that are printed in our Bibles that scholars have found strong evidence to suggest are late additions.

In fact, it might be a nice teaching opportunity, triggered when a quizzer asks, "Why do we skip these passages?" We can explain that the Bible we have is a printed book containing content copied from copies of copies of copies. We don't have the originals, but we have strong evidence to conclude that the vast majority of what we have is extremely close to what was likely the original.

**Comment by jswingle** [comment]
I would lean toward continuing to include it, simply because it is a larger chunk of verses (more than just one verse, like Romans 16:23) which have a lot of history in the church. It's important to know what this section says. I would also argue that there are no theological issues with the text, even though it's likely not Johannine in origin (interpreting Jesus' forgiveness of the woman as a license to sin is an obvious, and self-serving, misinterpretation...) it may not be Spirit-inspired, but it has a history in the church and it's not like we're asking them to randomly memorize the Gospel of Thomas or anything.

If we still did Mark, I would say we should include the longer ending as well.

That said, this is a "lean" of an opinion, I could be swayed otherwise. I think this issue is all about pedagogy - how can we best teach the Bible? - so my entire opinion would be based on what I think best does that.

**Comment by kclimenhaga** [comment]
I'm inclined to agree with Jeremy. I do not believe this passage to be heretical, even if it can be interpreted in heretical ways. If we were already omitting other parts of John, I agree that it would make sense to omit this passage, but since we are including the rest of John, I don't think we need to omit this one. That being said, I am also willing to be persuaded otherwise. 

The good news is that we aren't scheduled to quiz on John for several years yet, so we can take the time to gather lots of feedback on this topic.

**Comment by jswingle** [comment]
Another thing worth pointing out, although it is minor in the grand scheme of things, is that John is already a huge material year with plenty for the quizzers to study. It's not like we're taking 12 verses away from, say, Hebrews+Peter year. I think the strongest argument for getting rid of 7:53-8:11 is just that it's bracketed in the NIV and potentially could take away time from memorizing everything else.

**Comment by jswingle** [comment]
Revisiting this issue and I think I've changed my mind on the matter. Even though it's a longer section, there isn't really anything _in principle_ different from excluding Romans 16:23 since the NIV has it as a footnote. The only reason John 7:53-8:11 isn't in a footnote is that it's such a long section it wouldn't fit. They bracket the section instead.

Another argument that sways me in this direction is that we choose to quiz off the 2011 revision of the NIV. We should therefore go with whatever the 2011 NIV says -- not just in its translational choices, but also in its textual criticism choices. Once we've decided what translation to use, that's our material, _not_ the underlying manuscripts or original Greek language. We don't make rulings in the quiz room based off the Greek, after all. So if the 2011 NIV translators indicated that this is not Johannine in origin, we should exclude it from the quiz material whether we agree with them or not. (Likewise, if we were still quizzing on the KJV, I'd argue we should include the passage, even though I personally agree with the NIV translators' take.)

**Comment by jswingle** [comment]
Instead of the original proposal, however, I think we should include in the rulebook a master list of _all verses_ which are excluded from the material due to the textual critical choices of the NIV translators. 

My proposal is that we add a column to the chart in section "1.1 Material Rotation Schedule" with the title "Excluded Verses." I dug up a complete list of verses that should be indicated in this column:

Matthew 17:21, 18:11
Luke 17:36, 23:17
John 5:4, 7:53-8:11
Acts 8:37, 15:34
Romans 16:24

For ease of reading, we would notate these as simply "17:21, 18:11" without the title of the book for Matthew/Luke/John/Acts, and only use the book title for "Romans 16:24" since that year also includes James.

Lastly, under the chart we can include an explanatory comment also within section 1.1: 
`The "Excluded Verses" column lists verse numbers which are footnoted or bracketed in the NIV. These are not included in the quiz material.`

**Comment by JoshJetto** [comment]
The only reason I would disagree with this wording is that with the exception of John 7:53-8:11, all of these other verses are already omitted from the text in the NIV and not included in the quiz books. Unfortunately, because of its traditional standing, this section of John, though bracketed and noted as not being in the text of the Gospel of John in the earliest manuscripts, remains inserted in the main body of the NIV text. In other words, I don't think we need to draw attention to verses which only appear in the footnotes of the NIV for lack of textual support, but for this large section, which is not footnoted, we should note that we are skipping over it, because of the lack of textual support for it as inspired Scripture.

**Comment by jswingle** [comment]
@ARMediting I think if we succeeded in removing the numbers from the NIV, the Earth would rotate off its axis. It would be like dividing by zero. Can you even imagine people trying to look up a verse in John 8 and it's eleven verses up from where they thought it would be? Mass chaos. Congregations in disarray. Divisions and dissensions.

All this just for the sake of an easier quizzing ruleset? (Sure, I'm game. hahaha)

**Comment by josiah-leinbach** [comment]
As a historian I will note that even though the magisterial Reformers (rightly) did not consider the Jewish Apocrypha to be inspired Scripture, they still included it in their Bible translations on the grounds that it was an "example of life and instruction of manners" (following Jerome), and encouraged congregants to read the Jewish Apocrypha but **not** develop doctrine from it. The Luther Bible (1534), The Geneva Bible (1560), and Authorized Version (1611) all contain a translated apocrypha.

My point here is not to debate for or against the canonicity of the _Pericope Adulterae_ but rather to show that there is historic precedent we could draw from that would justify having quizzers study John 7:53-8:11, regardless what people's opinions of its canonicity are.

**Comment by jswingle** [comment]
I agree with everything in @josiah-leinbach 's post in principle, and if we had all the time in the world I would absolutely sponsor a quiz year on the Wisdom of Solomon because it's fantastic.

But, I think our material cycle should emphasize scriptural content itself. I would still recommend that local churches read the beginning of John 8 and talk about it, but we have to think about what the rules do or don't encourage at a broad level.

**Comment by 24RMiller** [comment]
I see the value in this passage, but I would be in favor of striking these 12 verses from our material on the basis of keeping consistency.

Without going down a theological or historical rabbit hole, every other footnoted verse is excluded from our quiz material. This passage is functionally the same. As to the spiritual value, I agree it's an important passage, but so are the other excluded verses. Acts 8:37 states: "Philip said, 'If you believe with all your heart, you may.' The eunuch answered, 'I believe that Jesus Christ is the Son of God.'" This is just as "spiritually significant" of a verse, yet we don't think twice about excluding it.

I do think clarifying that any verses/passages footnoted in the 2011 NIV are not to be in the quiz material would be useful as well.
